Assets/Scripts/GoalController.cs

```diff
@@ -14,6 +14,8 @@ private void OnTriggerEnter2D(Collider2D collision)
             ServiceLocator.DataCollector.SendColorSwitchCountsToFirebase();
             //ServiceLocator.DataCollector.SendLevelCompletionTimeToFirebase();
             ServiceLocator.DataCollector.ResetColorSwitchCounts();
+            ServiceLocator.DataCollector.SendZoneTimesToFirebase(); // Send all zone times here
```

Comment:
@alekhyar2511 and @qaisusc have made changes to the same function.
Organize your merges to main in such a way that no conflicts arise.

Check #21 for @alekhyar2511 's changes to this function.

Reply:
Handled @qaiisusc function call.